docs/parts/5-prism_central.md

```diff
@@ -0,0 +1,100 @@
+
+![Prism Central VM Tab](../images/prism-central-vm-tab.png)
+
+### 4. Log in to Prism Central and Change its Password
+
+Now we need to login to Prism Central. We recommend doing this in a separate web browser or a Private/Incognito window. You can access the Prism Central UI from your computer at `https://localhost:19440`. Note that the Prism Central UI ships with a self-signed TLS certificate, so most browsers will display a security warning. This security warning is unavoidable. If your browser will allow you to ignore the warning and load the site, do that; if not, try a different browser.
+
+Log in to the Prism UI using the default credentials [found here on the Nutanix Support Portal](https://portal.nutanix.com/page/documents/details?targetId=AHV-Admin-Guide-v6_7:app-admin-cvm-access-c.html). You will be forced to change the password when you first log in. After changing the password, you will be automatically logged out.
+
+### 4. Register the Prism Central VM
+
+Now that Prism Central is deployed, we need to register the cluster to it. Go back to the original Prism window and click on the gear icon in the upper right corner of the Prism UI and select `Prism Central Registration`. Click `Register or create new` and then click `Connect` under `I already have a Prism Central instance deployed`.
+
+![Prism Central Registration](../images/prism-central-registration.png)
+
+Click `Next` on the `Connect Info` page and then put in the Prism Central IP address `192.168.103.252`, port of `9440`, and the admin username and password you just set up for Prism Central.
+
+![Prism Central Registration Config](../images/prism-central-registration-config.png)
+
+After a few seconds, registration will complete. You will need to refresh the page to see the updated registration status. Now we can go login to Prism Central.
+
+![Prism Central Connected](../images/prism-central-connected.png)
+
+### 5. Configure Prism Central
+
+Let's start setting up some basic settings in Prism Central. So let's go back to the Prism Central window. Start by clicking the gear icon in the upper right corner of the Prism Central UI and selecting `Name Servers` from the settings menu on the left. Then add some public DNS servers to the list, we'll use `1.1.1.1` and `8.8.8.8`.
+
+![Name Servers](../images/name-servers.png)
+
+Next click on NTP Servers on the left and add a local NTP pool server for the region you deployed Nutanix into. We'll use `0.north-america.pool.ntp.org`
+
+![Prism Central NTP Servers](../images/prism-central-ntp-servers.png)
+
+In the next part we'll deploy another piece of the Nutanix stack onto the cluster.
+
+### 11. Wait for update to complete and continue to next Part
+
+In the next part we'll deploy another piece of the Nutanix stack onto the cluster.
```

Comment:
I didn't have to wait for anything after putting in the name servers and the NTP server.  Did I miss a step to upgrade something?

Reply:
Nope, I removed updating from this section and forgot to delete this line.